homeassistant/components/starlink/sensor.py

```diff
@@ -155,4 +155,10 @@ def native_value(self) -> StateType | datetime:
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
         value_fn=lambda data: data.consumption["total_energy"],
     ),
+    StarlinkSensorEntityDescription(
+        key="status",
```

Comment:
I don't like this generic name (and it's not even a status but a state).

I'd go w/ "connection" or maybe even better by `device_class`. 😉